### ISSUE TITLE
Simulators are not found, getting [ ]

### ISSUE THREAD

**Issue by billburgess** [wix/AppleSimulatorUtils#54]
**Description**
Running `applesimutils --list` is not returning a list of available simulators as expected.
Running `xcrun simctl list devices` returns a full list of available devices that should be found.

```
applesimutils --list
[

]
```

```
xcrun simctl list devices
== Devices ==
-- iOS 10.2 --
    iOS 10 Device (887E2288-59E0-43EF-AB5D-C6C759739452) (Shutdown) 
-- iOS 12.2 --
    iPhone SE (35ECF1A7-005D-426A-A0D5-6F0A365C92F5) (Shutdown) 
    iPhone X (99AA3344-59C2-4214-A612-F17381A06AF7) (Shutdown) 
    iPhone Xs Max (4D05EA4E-11F5-4FCB-8B25-9F280B743DB2) (Shutdown) 
    iPad Air 2 (7D82D4ED-606E-4B4C-83C8-22B1FBC0CA65) (Shutdown) 
    iPad Pro (11-inch) (DB1F8D5F-7F04-40AF-943B-DE32C8BE303F) (Shutdown) 
    iPadTestDevice (88FB61A2-9807-40D4-BB42-E8258970EFB1) (Shutdown) 
-- tvOS 12.2 --
    Apple TV 4K (38884A89-F573-4382-BEC8-663B910F1863) (Shutdown) 
-- watchOS 5.2 --
    Apple Watch Series 2 - 38mm (BA61EADE-C7C1-441E-B2C5-1F8E322F10CE) (Shutdown) 
    Apple Watch Series 2 - 42mm (7777B94E-BB97-433C-91C3-7EDF9F4A5818) (Shutdown) 
    Apple Watch Series 3 - 38mm (09BB2B06-38D1-4960-A5CE-F6AED3A355E8) (Shutdown) 
    Apple Watch Series 3 - 42mm (3048BA2E-BDBD-474A-9F81-8F2C431D2497) (Shutdown) 
-- Unavailable: com.apple.CoreSimulator.SimRuntime.iOS-13-0 --
    iPhone 8 (979F01E2-05CA-46E5-86BA-3AFFB361D86A) (Shutdown) (unavailable, runtime profile not found)
    iPhone 8 Plus (C24C424C-9563-466C-B031-1A56DB438A07) (Shutdown) (unavailable, runtime profile not found)
    iPhone Xs (D338FF86-A7A9-48BC-8AB2-5091EC48FA3E) (Shutdown) (unavailable, runtime profile not found)
    iPhone Xs Max (992D0848-961E-4110-A1B1-994AA1C24386) (Shutdown) (unavailable, runtime profile not found)
    iPhone Xʀ (CF17A29D-3A16-4E2C-8016-72A065AE42AF) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (9.7-inch) (9BB77090-9E62-4EAE-9789-E9FBBCD8B0EB) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (11-inch) (7490EBED-74AB-4E46-9B3F-61A8566663A7) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (12.9-inch) (3rd generation) (CDE08C47-DE72-4254-B556-C0605BEB5402) (Shutdown) (unavailable, runtime profile not found)
    iPad Air (3rd generation) (057F3AEA-A579-40E1-8752-8D0B93DBC512) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.tvOS-13-0 --
    Apple TV (0C4B643F-462E-41B1-809D-75C67C021C16) (Shutdown) (unavailable, runtime profile not found)
    Apple TV 4K (3328165F-C9C7-43E3-B691-1AA6E0D6348D) (Shutdown) (unavailable, runtime profile not found)
    Apple TV 4K (at 1080p) (D2AA75C4-0AB5-487F-8AFA-C31E4D666E4B) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.watchOS-6-0 --
    Apple Watch Series 2 - 38mm (74733A41-7197-4D6F-8FAC-97F8242172F1) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 2 - 42mm (B36F8F0D-2B47-4156-BAC1-9883BAEFA10B) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 3 - 38mm (AD07B3C7-2236-4024-8E8D-431B66EDDFA8) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 3 - 42mm (382B8DF2-2A63-437B-9271-4C73AB0C22D9) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 4 - 40mm (DBCD9524-999B-4019-8C96-E9410F3C589A) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 4 - 44mm (DE683917-1E3E-4163-8078-451E5C9A9C7A) (Shutdown) (unavailable, runtime profile not found)
```

**Environment**
- macOS version: 10.14.5
- Xcode version: 10.2.1
- Command Line Tools: 10.2.1
- applesimutils: 0.6.6

**Additional Context**
I have untapped and retapped per the pinned issue with 0.6.6. I uninstalled and reinstalled to get on the latest. I'm not sure what might be holding it back from seeing the list of simulators. My local scripts aren't working due to no simulators being found.

**Comment by LeoNatan** [comment]
Wow, I reproduced this. Investigating.

**Comment by LeoNatan** [comment]
So I managed to reproduced only after installing the Xcode 11 beta. Did you do the same?

**Comment by billburgess** [comment]
Yes. I have the beta installed.

**Comment by LeoNatan** [comment]
I think I remember something about there being only a single simulator runtime, and once Xcode is updated, the runtime goes up. So something must have changed in the format.

**Comment by billburgess** [comment]
Do you think removing Xcode 11 beta will fix the issue?

**Comment by LeoNatan** [comment]
I'm not sure. Still, I wouldn't expect you to. I am investigating.

**Comment by LeoNatan** [comment]
Found it. Will release a new version now.

**Comment by LeoNatan** [comment]
Please try 0.6.7

**Comment by billburgess** [comment]
Confirmed fixed in 0.6.7. Thanks for the quick turnaround.
